snap/quota/resources.go

```diff
@@ -203,14 +203,19 @@ func (qr *Resources) CheckFeatureRequirements() error {
 	return nil
 }
 
+// IsZero returns true if there is no limit set
+func (qr *Resources) IsZero() bool {
```

Comment:
Unset() bool might be preferable here?

Reply:
`Unset` implies action, I'd rather have IsUnset to clarify a boolean result - IsZero is used multiple places by golang.

Reply:
I'll change it to Unset so it matches our current convention, but my opinion is it's not logical for that operation.

And thanks for the review!